apps/site/assets/css/_customer-support.scss

```diff
@@ -169,3 +169,29 @@ button .waiting {
   top: -$base-spacing / 2;
   width: $base-spacing;
 }
+
+.c-expandable-block__link {
+  color: inherit;
+  text-decoration: none;
+  &:hover {
+    text-decoration: inherit;
+  }
+  &:link {
+    text-decoration: inherit;
+  }
+}
+
+
+a[aria-expanded='false'] > div > .c-expandable-block-caret {
+  @include caret($color: $brand-primary);
+  &::after {
+    content: $fa-var-angle-down;
+  }
+}
+
+a[aria-expanded='true'] > div > .c-expandable-block-caret {
```

Comment:
You probably don't need the `> div >` here

Reply:
Have you considered reusing the `.c-expandable-block__header-caret` class in the template instead of using `.c-expandable-block-caret`? Then you wouldn't need to `@include caret` again and can do something like this for rendering the caret:

```scss
.c-expandable-block__header-caret {
  [aria-expanded='false'] &::after {
    content: $fa-var-angle-down;
  }

  [aria-expanded='true'] &::after {
    content: $fa-var-angle-up;
  }
}
```

Reply:
Yes, this is simpler, thanks!